modules/grafana/files/dashboards/email_alert_api.json

```diff
@@ -10,56 +10,43 @@
         "name": "Email Alert API Deploys",
         "showIn": 0,
         "tags": "deploys email-alert-api",
-        "type": "alert"
+        "target": "",
+        "type": "event"
       },
       {
         "datasource": "Graphite",
         "enable": true,
         "hide": false,
-        "iconColor": "rgb(231, 255, 96)",
+        "iconColor": "rgba(255, 96, 96, 1)",
         "limit": 100,
         "name": "Email Alert API Restarts",
         "showIn": 0,
         "target": "substr(stats.govuk.app.email-alert-api.restarts,3,5)",
         "type": "alert"
-      },
-      {
-        "datasource": "Graphite",
-        "enable": true,
-        "hide": false,
-        "iconColor": "rgb(96, 177, 255)",
-        "limit": 100,
-        "name": "Email Alert Service Deploys",
-        "showIn": 0,
-        "tags": "deploys email-alert-service",
-        "type": "alert"
-      },
-      {
-        "datasource": "Graphite",
-        "enable": true,
-        "hide": false,
-        "iconColor": "rgb(96, 255, 107)",
-        "limit": 100,
-        "name": "Email Alert Service Restarts",
-        "showIn": 0,
-        "target": "substr(stats.govuk.app.email-alert-service.restarts,3,5)",
-        "type": "alert"
       }
     ]
   },
-  "editMode": false,
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
   "hideControls": false,
-  "id": 34,
+  "id": 141,
   "links": [],
-  "refresh": "10s",
+  "refresh": "1m",
   "rows": [
     {
       "collapse": false,
-      "height": "280",
+      "height": 239,
       "panels": [
+        {
+          "content": "   - [Sidekiq](https://grafana.publishing.service.gov.uk/dashboard/file/sidekiq.json?refresh=1m&orgId=1&var-Application=email-alert-api&var-Queues=All&from=now-3h&to=now)\n   - [Machine stats](https://grafana.publishing.service.gov.uk/dashboard/file/machine.json?refresh=1m&orgId=1&var-hostname=email-alert-api*&var-cpmetrics=cpu-system&var-cpmetrics=cpu-user&var-filesystem=All&var-disk=All&var-tcpconnslocal=All&var-tcpconnsremote=All)",
```

Comment:
Do we have precedent for this as production carrenza only links? could we just have absolute paths for portability?

Reply:
Another good spot - they're relative now 😸